src/Features/Http.php

```diff
@@ -1,13 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Blumilk\BLT\Features;
-
-use Behat\Behat\Context\Context;
-use Blumilk\BLT\Features\Traits\Http as HttpTrait;
-
-class Http implements Context
-{
-    use HttpTrait;
```

Comment:
Maybe leave this also, if it won't make to much clutter. Just add both traits to class.